components/profile/ProfileDetail.vue

```diff
@@ -699,7 +699,7 @@ const interactionIn = computed(() => {
 useAsyncData('tabs-count', async () => {
   const address = id.value.toString()
   const searchParams = {
-    currentOwner_eq: address,
+    currentOwner_containsInsensitive: address,
```

Comment:
currentOwner_eq: vm === 'EVM' ? address.toLowerCase() : address

Reply:
since we will be using this quite a lot.  let's make a util `toIndexerAddress` or any other name 
````
const toIndexerAddress =  (address:, {vm, prefix}) => {
    return pickByVm({
        EVM: address.toLowerCase(),
        SUB: address
    }, { vm , prefix })
}
````

Reply:
I am ok with any solution except contains eq ^-^

Reply:
> since we will be using this quite a lot. let's make a util `toIndexerAddress` or any other name
> 
> ```
> const toIndexerAddress =  (address:, {vm, prefix}) => {
>     return pickByVm({
>         EVM: address.toLowerCase(),
>         SUB: address
>     }, { vm , prefix })
> }
> ```

There is an edge that does not fit. It's possible to show evm address when you are on sub network. @hassnian 
I would make it like this:
 ```
 const toIndexerAddress =  address => isEthereumAddress(address) ? address.toLowerCase() : address;
```

Reply:
`profileTabsCount` still doesn't work with lower case address & `currentOwner_eq`, check graphql response before trying weird hack

Reply:
@roiLeo can you share more pls?

Reply:
```
query profileTabsCount($id: String!, $denyList: [String!]) {

  owned: nftEntitiesConnection(
    where: {
      currentOwner_eq: $id,
      burned_eq: false,
      metadata_not_eq: ""
      issuer_not_in: $denyList
     }
    orderBy: name_ASC
  ) {
    totalCount
  }

}
```

result:
```
{
  "data": {
    "owned": {
      "totalCount": 0
    }
  }
}
```

while containsInsensitive
```
query profileTabsCount($id: String!, $denyList: [String!]) {

  owned: nftEntitiesConnection(
    where: {
      currentOwner_containsInsensitive: $id,
      burned_eq: false,
      metadata_not_eq: ""
      issuer_not_in: $denyList
     }
    orderBy: name_ASC
  ) {
    totalCount
  }

}
```

result:
```
{
  "data": {
    "owned": {
      "totalCount": 39
    }
  }
}
```

Reply:
should be good now

![Screenshot 2024-09-30 at 12-26-00 NFT Artist Profile on KodaDot One Stop Shop for Polkadot NFTs](https://github.com/user-attachments/assets/7cd63c5c-8c2d-4eb1-a274-e92e1f53ff60)